src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+
+  return styles
+    .map((item) => item.split(':'))
```

Comment:
The current implementation splits each style by the first colon, which may not correctly handle values containing colons. Consider using a more robust method to handle cases where property values contain colons, as this is a common scenario in CSS (e.g., `url(http://example.com)`).